### ISSUE TITLE
Problem with eslint v8 - Failed to load plugin 'gridsome'

### ISSUE THREAD

**Issue by tennox** [gridsome/eslint-plugin-gridsome#143]
**Tell us about your environment**

- **ESLint version:** 8.25.0
- **eslint-plugin-gridsome version:** 1.6.2
- **Node version:** 16

**Please show your full configuration:**

```javascript
module.exports = {
  parser: 'vue-eslint-parser',
  extends: [
    'standard',
    'plugin:gridsome/recommended',
    'plugin:vue/vue3-recommended'
  ],
}

```

**What did you do?**

`eslint --ext .js,.vue src/file.vue`

**What actually happened?**

```
Oops! Something went wrong! :(

ESLint: 8.26.0

TypeError: Failed to load plugin 'gridsome' declared in '.eslintrc.js': Class extends value undefined is not a constructor or null
Referenced from: /home/manu/dev/project/.eslintrc.js
    at Object.<anonymous> (/home/manu/dev/project/node_modules/.pnpm/@typescript-eslint+experimental-utils@3.10.1_wyqvi574yv7oiwfeinomdzmc3m/node_modules/@typescript-eslint/experimental-utils/dist/ts-eslint/CLIEngine.js:12:34)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/manu/dev/project/node_modules/.pnpm/@typescript-eslint+experimental-utils@3.10.1_wyqvi574yv7oiwfeinomdzmc3m/node_modules/@typescript-eslint/experimental-utils/dist/ts-eslint/index.js:14:14)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
 ELIFECYCLE  Command failed with exit code 2.
```

Downgrade to v7 fixes the problem

**Comment by tyankatsu0105** [comment]
Thanks for the issue.
This plugin has not support ESLint v8 yet.
We just do this problem.
So please wait until we finish do that.
